x-pack/legacy/plugins/siem/public/components/ml/tables/anomalies_network_table.tsx

```diff
@@ -56,40 +56,43 @@ export const AnomaliesNetworkTable = React.memo<AnomaliesNetworkTableProps>(
       pageSizeOptions: [5, 10, 20, 50],
       hidePerPageOptions: false,
     };
+
     if (!hasMlUserPermissions(capabilities)) {
       return null;
     } else {
       return (
-        <EuiPanel>
-          <BasicTableContainer>
-            {loading && (
-              <>
-                <BackgroundRefetch />
-                <LoadingPanel
-                  height="100%"
-                  width="100%"
-                  text={`${i18n.LOADING} ${i18n.ANOMALIES}`}
-                  position="absolute"
-                  zIndex={3}
-                  data-test-subj="anomalies-network-table-loading-panel"
-                />
-              </>
-            )}
-            <HeaderPanel
-              subtitle={`${i18n.SHOWING}: ${networks.length.toLocaleString()} ${i18n.ANOMALIES}`}
-              title={i18n.ANOMALIES}
-              tooltip={i18n.TOOLTIP}
-            />
-            <BasicTable
-              items={networks}
-              columns={columns}
-              pagination={pagination}
-              sorting={sorting}
-            />
-          </BasicTableContainer>
-        </EuiPanel>
+        <Panel loading={loading}>
```

Comment:
it seems really familiar to anomalies_host_table maybe we can create a generic component that they can share instead of duplicating code

Reply:
Yeah, I agree. When speaking to @FrankHassanabad about the anomaly tables though, he indicated that further changes and enhancements were coming for these tables.

Frank, would consolidation of these two tables into a single component be something you're planning on tackling? Or do you think I take a stab at adding it to this PR?